Packs/Zimperium/Integrations/ZimperiumV2/ZimperiumV2.py

```diff
@@ -0,0 +1,1034 @@
+
+    Returns:
+        Outputs.
+    """
+    page = arg_to_number(args.get('page', '0'))
+    page_size = arg_to_number(args.get('page_size'))
+    limit = arg_to_number(args.get('limit', '50'))
+    team_id = args.get('team_id')
+
+    size = page_size if page_size else limit
+
+    response = client.policy_device_inactivity_list(size=size, page=page, team_id=team_id)
+
+    hr = tableToMarkdown(name='Device Inactivity List', t=response,
+                         headers=['id', 'name', 'teamId'],
+                         headerTransform=pascalToSpace)
+
+    command_results = CommandResults(
+        outputs_prefix='Zimperium.PolicyDeviceInactivity',
+        outputs=response,
+        outputs_key_field='id',
+        readable_output=hr,
+        raw_response=response,
+    )
+    return command_results
+
+
+def policy_device_inactivity_get_command(client: Client, args: dict) -> CommandResults:
+    """Get device inactivity policy by id.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs.
+    """
+    policy_id = args.get('policy_id')
+
+    response = client.policy_device_inactivity_get(policy_id=policy_id)
+
+    hr = tableToMarkdown(name='Device Inactivity', t=response,
+                         headers=['id', 'name', 'teamId', 'pendingActivationSettings',
+                                  'inactiveAppSettings', 'created', 'modified',
+                                  ],
+                         headerTransform=pascalToSpace,
+                         removeNull=True,
+                         date_fields=['created', 'modified']
+                         )
+
+    command_results = CommandResults(
+        outputs_prefix='Zimperium.PolicyDeviceInactivity',
+        outputs=response,
+        outputs_key_field='id',
+        readable_output=hr,
+        raw_response=response,
+    )
+    return command_results
+
+
+def fetch_incidents(client: Client, last_run: dict, fetch_query: str,
+                    first_fetch_time: Optional[str], max_fetch: int, look_back: int = 1) -> tuple[list, dict]:
+    """
+    This function will execute each interval (default is 1 minute).
+
+    Args:
+        client (Client): Zimperium V2 client.
+        last_run (dict): the last fetch object.
+        fetch_query(str): fetch query to search.
+        first_fetch_time (time): If last_run is None then fetch all incidents since first_fetch_time.
+        max_fetch(int): max events to fetch.
+        look_back(int): minutes to look back when fetching.
+
+    Returns:
+        next_run: This will be last_run in the next fetch-incidents
+        incidents: Incidents that will be created
+    # """
+    demisto.debug(f"Last run before the fetch run: {last_run}")
+    limit = last_run.get('limit', max_fetch)
+    start_time, end_time = get_fetch_run_time_range(
+        last_run=last_run,
+        first_fetch=first_fetch_time,
+        look_back=look_back,
```

Comment:
so we are using the look back because there is an indexing issue in the 3rd party?